src/DetectChanges/BCBreak/ClassBased/MethodChanged.php

```diff
@@ -48,10 +45,10 @@ private function methods(ReflectionClass $class): array
     {
         $methods = $class->getMethods();
 
-        return array_combine(
-            array_map(static function (ReflectionMethod $method): string {
-                return strtolower($method->getName());
-            }, $methods),
+        return Dict\associate(
+            Vec\map($methods, static function (ReflectionMethod $method): string {
```

Comment:
Similar to other comments about `array_map()`, I would suggest keeping this as it was